### PR TITLE
Adding r-repurrrsive recipe

### PR DESCRIPTION
**Pull request by chameroy**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by chameroy** [comment]
Sorry I haven't been able to get this right. If this one does go through, then I'll just leave it to the pros. I had also created an issue requesting this package.

Thank you for your patience.

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for adding this. This is missing a license file (we require this). We generally want this to come from the tarball, but there is no such file. Additionally, upstream does not have an explicit LICENSE file. Technically, this is CC0, so we could do anything we want (add our own copy), but I'd prefer we first try communicating with upstream. Specifically, I'd suggest asking upstream that they:

1. Include an explicit copy of CC0 as "LICENSE" in the repository.
2. Package this in their releases.

See if they're open to this; if not package a copy of CC0.

**Comment by chameroy** [comment]
Where would I get the information for the LICENSE.txt file from? Would I just copy it from the meta.yaml file?

**Comment by mfansler** [comment]
The text can be gotten from Creative Commons or SPDX. E.g., https://github.com/spdx/license-list-data/blob/main/text/CC0-1.0.txt

It doesn't need any specific adjustment, just added as a `LICENSE` (no .txt) file.

File: recipes/r-repurrrsive/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+about:
+  home: https://jennybc.github.io/repurrrsive/, https://github.com/jennybc/repurrrsive
+  license: CC0-1.0
+  summary: Recursive lists in the form of R objects, 'JSON', and 'XML', for use in teaching and
+    examples. Examples include color palettes, Game of Thrones characters, 'GitHub'
+    users and repositories, music collections, and entities from the Star Wars universe.
+    Data from the 'gapminder' package is also included, as a simple data frame and in
+    nested and split forms.
+  license_family: CC
+
```
Comment (mfansler) [inline recipes/r-repurrrsive/meta.yaml:48]: Need to package license.

```suggestion
  license_file:
    - LICENSE

```

**Review by mfansler — CHANGES_REQUESTED** [review]

**Comment by chameroy** [comment]
Thank you for your patience while I get the hang of this. Hopefully, this does it?

**Review by mfansler — APPROVED** [review]
Looks good! Thanks again for contributing this!

**Comment by chameroy** [comment]
Is there anyway to make a (hopefully) final adjustment? Installing the package wants to downgrade r-base 4.3.1 -> 4.2.3. Going forward, I'll be sure to specify the R version.

**Comment by mfansler** [comment]
[R 4.3 is still migrating](https://conda-forge.org/status/#r_base43), i.e., not part of our default build set yet. The feedstock should get a PR to add an R 4.3 build sometime today. This is not something you need to specify in a recipe submitted to **staged-recipes**.

**Comment by chameroy** [comment]
Good to hear. When a new version of R come out, about how long, on average, does it take for it to get added to the feedstock?

**Comment by mfansler** [comment]
It really depends. Roughly a few weeks *once we start migrating*, which is kinda the wildcard.

Last time, there was extensive delay (months) getting `r-base=4.2` built. This time `r-base=4.3` built the next day, but took 5 weeks to kick off migration (uncertainty around Windows fate). But the migrating was 2-3 weeks to get through the vast majority of feedstocks.

**Pull request merged**